lib/travis/api/v3/remote_query.rb

```diff
@@ -56,7 +56,16 @@ def config
     end
 
     def s3_config
-      raise NotImplemented
+      conf = config.log_options.try(:s3) || {}
```

Comment:
I'll revert to what we had before. The connection error, now I think about it, might have been because logs_options didn't exist in the heroku config (only log_options did)